pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -937,7 +937,20 @@ self: super: {
 
   # Generate cli completions for dhall.
   dhall = generateOptparseApplicativeCompletion "dhall" super.dhall;
-  dhall-json = generateOptparseApplicativeCompletions ["dhall-to-json" "dhall-to-yaml"] super.dhall-json;
+  dhall-json =
+    let
+      tastyHunitWithOverrides = self.tasty-hunit.override {
+        tasty = pkgs.haskellPackages.tasty_1_4_0_2;
+      };
+
+      dhallJsonWithOverrides = super.dhall-json.override {
+        tasty-hunit = tastyHunitWithOverrides;
+        tasty-silver = self.tasty-silver.override {
+          tasty-hunit = tastyHunitWithOverrides;
+        };
```

Comment:
@cideM I'm not really sure what to do here.

Maybe you could move this `tasty-silver` override to the top-level in this file?  I don't think it makes sense to have `tasty-silver` _only_ working for `dhall-json`.

Alternatively, you could use `overrideScope` on `dhall-json` so all you have to do is override the `tasty` version, and that gets automatically propogated to `tasty-silver` and `tasty-hunit`.

Reply:
Oh, also, just disabling tests for `dhall-json` might be the easiest thing to do, since then it probably wouldn't pull in any of the tasty dependencies.

You could just link to commercialhaskell/stackage#5795 in the code.